pkg/api/http_server.go

```diff
@@ -406,45 +413,56 @@ func (hs *HTTPServer) Run(ctx context.Context) error {
 	hs.log.Info("HTTP Server Listen", "address", listener.Addr().String(), "protocol",
 		hs.Cfg.Protocol, "subUrl", hs.Cfg.AppSubURL, "socket", hs.Cfg.SocketPath)
 
-	var wg sync.WaitGroup
```

Comment:
Note for reviewers: This was copied from https://github.com/grafana/grafana/pull/67945/files?diff=unified&w=1#diff-059f97ccda689ebea0e2f9981cd736d77b1fa01a719d95abae333ad40fb0b9c4L414, please double check this. 

Though I can say it appears to be working :
```
INFO [07-13|15:34:10] module stopped                           logger=modules module=background-services
INFO [07-13|15:34:10] All modules stopped                      logger=modules modules="[background-services http-server]"
```

Reply:
another way to test is to add this to your `custom.ini`:

```
target = background-services
```

that should switch from the default `target = all` and prevent the http server from starting

Reply:
Perfect! I verified both, worked as expected (neat!)